RELEASE.md

```diff
@@ -4,15 +4,20 @@ Polkadot Release Process
 ### Branches
 * release-candidate branch: The branch used for staging of the next release.
   Named like `release-v0.8.26`
-* release branch: The branch to which successful release-candidates are merged
-  and tagged with the new version. Named literally `release`.
-
+  
 ### Notes
 * The release-candidate branch *must* be made in the paritytech/polkadot repo in
 order for release automation to work correctly
 * Any new pushes/merges to the release-candidate branch (for example,
-refs/heads/release-v0.8.26) will result in the rc index being bumped (e.g., v0.8.26-rc1
-to v0.8.26-rc2) and new wasms built.
+  refs/heads/release-v0.8.26) will result in the rc index being bumped (e.g., v0.8.26-rc1
+  to v0.8.26-rc2) and new wasms built. If you need to merge several changes into
+a new release-candidate, it's advised to do this as a single PR against the rc branch
+* There is a step on the auto-created github issue checklist that specifies we should
```

Comment:
Would that now be better to update the mentioned step instead of having contradictory info here ?